### PR TITLE
Generate unique id for each node

### PR DESCRIPTION
**Pull request by dgliwka**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -58,36 +60,33 @@ export default class App extends Component<{}, { chosenNode: TreeNode<string>, v
...
+            chosenNode: this.state.ourTree
         })
     }
 
-    displayNode(event: any, node_key: string) 
+    selectNode(event: any, node_id: string) 
     {
         this.setState({
-            chosenNode: this.state.ourTree.filter(n => n.data === node_key)[0]
+            chosenNode: node_id === this.state.ourTree.id ? this.state.ourTree : this.state.ourTree.filter(n => n.id === node_id)[0]
```
Comment (allgreed) [inline src/App.tsx]: What does the check here is for?

Comment (allgreed) [reply src/App.tsx]: bump

**Comment by allgreed** [comment]
Tried running the app as is, but fails to start due to linting errors, though no linting errors are actually found - removing `EXTEND_ESLINT=true` from `run` target seams to do the job

File: src/App.tsx (hunk 1 above)
Comment (dgliwka) [reply src/App.tsx]: Earlier version did not catch the edge case of root node.

**Pull request merged**